content/sensu-core/2.0/getting-started/configuring-sensuctl.md

```diff
@@ -191,6 +191,48 @@ sensuctl provides `import` commands for creating & updating resources via STDIN.
 OK
 {{< /highlight >}}
 
+## Create
+
+Similar to the `import` command, `sensuctl create` allows you to create and/or
+update resources by reading from STDIN or a flag configured file (`-f`). The
+accepted format of the `create` command is `wrapped-json`, which wraps the
+contents of the resource in `spec` and identifies its 2.x `type` (see below for
+an example, and [typemap][2] for a list of types).
```

Comment:
I think we should probably document this elsewhere. It's kind of unfriendly to refer users to source code.

Reply:
@echlebek Do you mean document this "Create" section elsewhere? Or document the list of types (rather than pointing to typemap) elsewhere?

Reply:
Yeah I think we should explicitly list the available types!